docs/release-notes.md

```diff
@@ -0,0 +1,64 @@
+---
+sidebar_position: 11
+id: release-notes
+description: prophecy monthly release notes
```

Comment:
I think we should add a folder Change log/Release notes

And each version as a new file.